### ISSUE TITLE
Demo Meeting April 21 - Material Sample Naming Discussions

### ISSUE THREAD

**Issue by ssbilkhu** [AAFC-BICoE/dina-planning#326]
Material Sample Naming Discussions, power point slides from meeting to follow

**Comment by ssbilkhu** [comment]
[DINA_naming_discussion_v0.pptx](https://github.com/AAFC-BICoE/dina-planning/files/8534721/DINA_naming_discussion_v0.pptx)

**Comment by ssbilkhu** [comment]
Notes from Meeting: 

1) How do we handle going from old naming to new naming for new generations or DNA extractions

2) Check to see if downstream phylogeny programs work with dashes


Action Items 
-----------------
@rintoult et all to create multiple generations diagram using proposed naming 
@banchinic to do the same for CCAMF

**Comment by banchinic** [comment]
Hi!

Here's my multiple generations example diagram. 
[CCAMF Generations.pdf](https://github.com/AAFC-BICoE/dina-planning/files/8543527/CCAMF.Generations.pdf)
I am not sure how to go about naming batches and vials in my case. I would really like to be able to group them as a batch in DINA's sample view (they were created from the same petri plate, on the same day and using the same method) since it would help make things clearer in the view (I don't need to see the 100 vials I have but would like to know the date of the most recent batch).

I have no good suggestion on naming batches except that I would like to see the date it was created in the sample view. I am open to ideas!

I need to stick with my naming system of incrementing the vials though because of the way the vials are made. There are already many steps under the laminar flow hood and I can't add more steps / change things. If you want more info on why I can explain it at our next meeting.

I have no input for DNA extractions since we do not have any of our DNA samples in DINA right now so I trust the CCFC's judgment on that and will support their point of view.

**Comment by rintoult** [comment]
Emily and I discussed our options for 1.5 hours today, we didn't come out of the discussion satisfied but at least now have looked at all the options we have thought of. Here is what we reviewed, the pitfalls of each and some questions we have.
[CCFC Material Sample Naming Conventions.pdf](https://github.com/AAFC-BICoE/dina-planning/files/8557849/CCFC.Material.Sample.Naming.Conventions.pdf)

**Comment by rintoult** [comment]
The circularity of deciding on name conventions with how they will be displayed in summary views has been nagging at me. Could we possibly see some mock ups of how it is predicted summary views based on searches will be displayed? I feel like most of the time I am assuming how I think it would happen but would like to know and comment on what is actually in the plan. 

For example - is it going to work like an internet ordering site, I search for a term, say species and it will show me there are 491 unique DAOMC numbers. 
Like this: 
Trichoderma harzianum (491)
DAOMC252433, 252432, 252427 etc

Then if I want to search on DAOMC 252433 it will show me that there are 19 material samples like this:
DAOMC 252433 (19)

Then what happens if I want to only see those that are freeze dried?
What if I want to see DNA extracts only?
What if from the species list I only want to see those from Canada?
If I search a range of identifiers, how will those results be displayed just the parent records and then I can go into more details with the actual copies?
These questions go on and on.

We have asked this a thousand times but for me these views are absolutely linked to how our naming works in concert with the search. We really need to know what the plan is for summary views.

Do other collection managers agree?

Tara

**Comment by banchinic** [comment]
Yes I totally agree with you Tara. This is very important and will basically decide if DINA is great and easy to use or if I am getting lost in all the samples all the time. I've had this concern from the start as well and I've voiced it many times. With everything being a material sample (instead of having DNA samples, Specimens and Specimens Replicates like in SeqDB) it could get overloaded very fast if the summary view isn't very well made and clear.
